src/plotApplications/components/map/TargetListLayer.js

```diff
@@ -0,0 +1,78 @@
+// @ flow
```

Comment:
The majority of this file is the same code as the other layer components (from the leases view). I guess we can do it this way since there's already a precedent, even if I'd really want to write an auxiliary component in between that only takes the popup content and handles everything else by itself.